build.gradle

```diff
@@ -77,22 +76,8 @@ def buildVersion = buildNumber != null ? buildNumber : 123
 def artifactName = "sidebar-sdk"
 def fullVersion = "${project.version}.${buildVersion}"
 def isReleaseBuild = !fullVersion.contains('SNAPSHOT')
-def isMasterBranch = false
-
-task getIsMasterBranch {
-	def branch = ""
-	def proc = "git rev-parse --abbrev-ref HEAD".execute()
-	proc.in.eachLine { line -> branch = line }
-	proc.err.eachLine { line -> println line }
-	proc.waitFor()
-	if (branch.startsWith('main')) {
-		isMasterBranch = true
-	} else {
-		isMasterBranch = false
-	}
-}
 
-task createVersionPropertiesFile {
+tasks.register('createVersionPropertiesFile') {
```

Comment:
why do we need this gradle task and this "versionJavaSDK.properties" file at all? We do have a property called "currentVersion" in the gradle.properties file, so isn't this redundant?

Reply:
This version is displayed in the Sidebar